scu_builders.py

```diff
@@ -326,7 +326,6 @@ def generate_scu_files(max_includes_per_scu):
     process_folder(["modules/openxr"], ["register_types"])
     process_folder(["modules/openxr/action_map"])
     process_folder(["modules/openxr/editor"])
-    process_folder(["modules/openxr/extensions"])
```

Comment:
Maybe keep it commented out, with a comment like:
```
#process_folder(["modules/openxr/extensions"])  # Sensitive include order for platform code.
```
to avoid someone trying to see which (sub)folders are missing in the list and adding it back.